api.md

```diff
@@ -2923,67 +2927,111 @@ dex | orderby_asc | orderby_nullsfirst
-Shows information about reorder policies that have been created by the user.
-(See [add_reorder_policy](#add_reorder_policy) for more information about
-reorder policies).
-
-
-#### Available Columns [](timescaledb_information-reorder_policies-available-columns)
 
-|Name|Description|
-|---|---|
-| `hypertable` | (REGCLASS) The name of the hypertable on which the policy is applied |
-| `index` | (NAME) Chunks fully older than this amount of time will be dropped when the policy is run |
-| `job_id` | (INTEGER) The id of the background job set up to implement the reorder policy|
-| `schedule_interval` | (INTERVAL)  The interval at which the job runs |
-| `max_runtime` | (INTERVAL) The maximum amount of time the job will be allowed to run by the background worker scheduler before it is stopped |
-| `max_retries` | (INTEGER)  The number of times the job will be retried should it fail |
-| `retry_period` | (INTERVAL) The amount of time the scheduler will wait between retries of the job on failure |
-
-#### Sample Usage [](timescaledb_information-reorder_policies-examples)
-
-Get information about reorder policies.
-```sql
-SELECT * FROM timescaledb_information.reorder_policies;
-
-     hypertable   |     hypertable_index_name     | job_id | schedule_interval | max_runtime | max_retries | retry_period
---------------------+-----------------------------+--------+-------------------+-------------+-------------+--------------
-     conditions   | conditions_device_id_time_idx |   1000 | @ 4 days          | @ 0         |          -1 | @ 1 day
-(1 row)
+## timescaledb_information.jobs [](timescaledb_information-jobs)
+Shows information about all jobs registered with the automation framework. 
+
+#### Available Columns [](timescaledb_information-jobs-available-columns)
+
+|Name|Description|
+|---|---|
+|`job_id` | (INTEGER) The id of the background job |
+|`application_name` | (NAME) Name of the policy or user defined action |
+|`schedule_interval` | (INTERVAL)  The interval at which the job runs |
+|`max_runtime` | (INTERVAL) The maximum amount of time the job will be allowed to run by the background worker scheduler before it is stopped |
+|`max_retries` | (INTEGER)  The number of times the job will be retried should it fail |
+|`retry_period` | (INTERVAL) The amount of time the scheduler will wait between retries of the job on failure |
+|`proc_schema` | (NAME) Schema name of the function or procedure executed by the job |
+|`proc_name` | (NAME) Name of the function or procedure executed by the job |
+|`owner` | (NAME) Owner of the job |
+|`scheduled` | (BOOLEAN) | Is the job scheduled to run automatically? |
+|`config` | (JSONB) | Configuration passed to the function specified by `proc_name` at execution time |
+|`next_start` | (TIMESTAMP WITH TIME ZONE) | Next start time for the job, if it is scheduled to run automatically |
+|`hypertable_schema` | (NAME) Schema name of the hypertable. NULL, if this is a user defined action.|
```

Comment:
also NULL for telemetry

Reply:
Not sure if we want to call out Telemetry specifically. 
```
|`hypertable_schema` | (NAME) Schema name of the hypertable. NULL, if **this is Telemetry or a user defined action **.|
```